src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.cpp

```diff
@@ -53,20 +53,30 @@ llvm::SmallVector<int64_t> createInt64VectorFromIndexExpr(
   return literalVector;
 }
 
+Value getTosaConstTensorSingleF32(
+    PatternRewriter &rewriter, Operation *op, float val) {
+  auto constType = RankedTensorType::get({}, rewriter.getF32Type());
+  auto constAttr = DenseElementsAttr::get(constType, val);
+
+  auto constOp =
+      rewriter.create<mlir::tosa::ConstOp>(op->getLoc(), constType, constAttr);
+  return constOp.getResult();
+}
+
 mlir::RankedTensorType reduceAxisToOne(llvm::ArrayRef<int64_t> shape,
     mlir::Type elementType, mlir::Attribute encoding) {
   return mlir::RankedTensorType::get(
       llvm::SmallVector<int64_t, 4>(shape.size(), 1), elementType, encoding);
 }
 
 mlir::ElementsAttr getElementsAttrFromConst(mlir::Value &val) {
-    if (auto source = val.getDefiningOp<mlir::ONNXConstantOp>()) {
-      if (source.value())
-        return source.value().value(); 
-    }
-    // if the constant is not an onnx.const it has to be a tosa.const
-    assert(val.getDefiningOp<mlir::tosa::ConstOp>());
-    return tosa::getValueFromTosaConst<ElementsAttr>(val);
+  if (auto source = val.getDefiningOp<mlir::ONNXConstantOp>()) {
```

Comment:
For code that is related to upstream, please only change formatting when you are editing these lines of code, or as a separate PR that only reformats (and do this upstream first to avoid merge issues).

There are some other places in the PR with gratuitous format changes - please also adjust these.

(And apply as a fixup/squash to avoid the reformatting being in the git history.)

Reply:
Absolutely. Thanks